Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -208,7 +208,7 @@ public override IMixedRealityController Controller
 
                 if (base.Controller != null && this != null)
                 {
-                    pointerName = gameObject.name;
+                    PointerName = gameObject.name;
```

Comment:
I think this + https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7012#discussion_r365006262 will now cause pointer GameObjects to be named correctly but anything that's a ControllerPoseSynchronizer but not a pointer to not update their name

Reply:
See other comment